frontend/src/routes/categories/[category_id]/+page.svelte

```diff
@@ -38,19 +48,61 @@
                 <BreadcrumbItem href="/categories">Conferences</BreadcrumbItem>
                 <BreadcrumbItem>{mocks[data.category_id - 1].name}</BreadcrumbItem>
             </ResponsiveBreadCrumb>
-            <Heading tag="h2">{mocks[data.category_id - 1].name}</Heading>
+            <Heading tag="h2">
+                {mocks[data.category_id - 1].name}
+            </Heading>
             <Heading tag="h6">
                 <Secondary>Review Deadline: {mocks[data.category_id - 1].deadline}</Secondary>
             </Heading>
         </div>
-        {#if mocks[data.category_id - 1].type === "External" && mocks[data.category_id - 1].is_my_category() }
-            <Button size="lg" color="primary" class="mx-auto lg:m-0"
-                    on:click={() => show_assign_modal = true}>Assign Reviewers
-            </Button>
+
+
+        {#if mocks[data.category_id - 1].type === "External"}
+            {#if mocks[data.category_id - 1].is_my_category()}
+                <Button size="lg"
+                        color="primary"
+                        class="mx-auto lg:m-0"
+
+                        on:click={mocks[data.category_id - 1].open ? (() => show_assign_modal = true) : ("")}
+                        href={(mocks[data.category_id - 1].open) ?
```

Comment:
if the category is open, the ExternalAssignModal is opened before then going to the href link ( so it flashed shortly)